### PR TITLE
OSDOCS 10733: Added a note to specify the EUS versions in ImageSetConfiguration file

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OSDOCS-10733

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/oc-mirror-enclave-support.adoc [hunk 1]
```diff
@@ -102,6 +102,13 @@ $ export UPDATE_URL_OVERRIDE=https://<osus.enterprise.in>/graph
 +
 For more information on setting up OSUS on an OpenShift cluster, see "Updating a cluster in a disconnected environment using the OpenShift Update Service".
 
+[NOTE]
+====
+When upgrading {product-title} Extended Update Support (EUS) versions, an intermediate version may be needed between the current and target versions. For example, if the current version is `4.14` and target version is `4.16`, you may need to include a version like `4.15.8` in the `ImageSetConfiguration` when using oc-mirror plugin v1.
```
Comment (kasturinarra) [inline modules/oc-mirror-enclave-support.adoc:107]: Enclave support means v2 right, why do we mention about v1 here ?

Comment (subhtk) [reply modules/oc-mirror-enclave-support.adoc:107]: It is v2, I will change it.

File: modules/oc-mirror-image-set-config-examples.adoc [hunk 2]
```diff
@@ -268,3 +268,35 @@ mirror:
...
+[discrete]
+[id="oc-mirror-image-set-examples-eus_{context}"]
+== Use case: Including the upgrade path for EUS releases
+
+The following `ImageSetConfiguration` file includes the `eus-x.xx` channel, where the `maxVersion` (e.g., `4.14`) is at least two minor versions higher than the `minVersion` (e.g., `4.12`).
+
+For example, in this `ImageSetConfiguration` file, the `minVersion` is set to `4.12.28`, while the `maxVersion` for the `eus-4.14` channel is `4.14.16`. 
+
+.Example `ImageSetConfiguration` file
+[source,yaml,subs=attributes+]
```
Comment (adellape) [inline modules/oc-mirror-image-set-config-examples.adoc:282]:
```suggestion
[source,yaml,subs="attributes+"]
```

File: modules/oc-mirror-image-set-config-examples.adoc [hunk 3]
```diff
@@ -268,3 +268,35 @@ mirror:
...
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    graph: true # Required for the OSUS ooperator
```
Comment (adellape) [inline modules/oc-mirror-image-set-config-examples.adoc:288]:
```suggestion
    graph: true # Required for the OSUS Operator
```

File: modules/oc-mirror-enclave-support.adoc [hunk 4]
```diff
@@ -102,6 +102,13 @@ $ export UPDATE_URL_OVERRIDE=https://<osus.enterprise.in>/graph
 +
 For more information on setting up OSUS on an OpenShift cluster, see "Updating a cluster in a disconnected environment using the OpenShift Update Service".
 
+[NOTE]
+====
+When upgrading {product-title} Extended Update Support (EUS) versions, an intermediate version may be needed between the current and target versions. For example, if the current version is `4.14` and target version is `4.16`, you may need to include a version like `4.15.8` in the `ImageSetConfiguration` when using oc-mirror plugin v2.
+
+oc-mirror plugin v2 might not always detect this automatically, so check the link:https://access.redhat.com/labs/ocpupgradegraph/update_path[Cincinnati graph web page] to confirm any required intermediate versions and add them manually to your configuration.
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/oc-mirror-enclave-support.adoc:109]: Enclave support if oc-mirror plugin v2. So there is a note for both v1 and v2.

**Review by tmalove — APPROVED** [review]

**Pull request merged**